rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/PropertiesServiceV2.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2016-2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.rest.service.v2;
+
+import uk.gov.gchq.gaffer.rest.ServiceConstants;
+
+import javax.ws.rs.core.Response;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * An implementation of {@link IPropertiesServiceV2} that gets the configured system properties
+ */
+public class PropertiesServiceV2 implements IPropertiesServiceV2 {
+    private static final Map<String, String> PROPERTIES = new HashMap<>();
```

Comment:
I think it might be better to not cache the properties in case any of the properties change over time. In the future, if we find it is really slow then we could consider adding a daily cache or something similar.